src/Tizen.NUI/src/public/Layouting/LinearLayout.cs

```diff
@@ -134,7 +177,61 @@ public Size2D CellPadding
         /// [Draft] Get/Set the alignment in the layout
         /// </summary>
         /// <since_tizen> 6 </since_tizen>
-        public LinearLayout.Alignment LinearAlignment { get; set; } = Alignment.Top;
+        public LinearLayout.Alignment LinearAlignment
+        {
+            get
+            {
+                return linearAlignment;
+            }
+
+            set
+            {
+                if (linearAlignment == value)
+                {
+                    return;
+                }
+
+                linearAlignment = value;
+
+                switch (linearAlignment)
+                {
+                    case Alignment.Begin:
+                        LinearHorizontalAlignment = HorizontalAlignment.Begin;
+                        break;
+                    case Alignment.End:
+                        LinearHorizontalAlignment = HorizontalAlignment.End;
+                        break;
+                    case Alignment.CenterHorizontal:
+                        LinearHorizontalAlignment = HorizontalAlignment.Center;
+                        break;
+                    case Alignment.Top:
+                        LinearVerticalAlignment = VerticalAlignment.Top;
+                        break;
+                    case Alignment.Bottom:
+                        LinearVerticalAlignment = VerticalAlignment.Bottom;
+                        break;
+                    case Alignment.CenterVertical:
+                        LinearVerticalAlignment = VerticalAlignment.Center;
+                        break;
+                    case Alignment.Center:
+                        LinearHorizontalAlignment = HorizontalAlignment.Center;
+                        LinearVerticalAlignment = VerticalAlignment.Center;
+                        break;
+                    default:
+                        break;
+                }
+            }
+         }
+
+        /// <summary>
+        /// Get/Set the horizontal alignment in the layout
+        /// </summary>
+        public LinearLayout.HorizontalAlignment LinearHorizontalAlignment { get; set; } = HorizontalAlignment.Begin;
```

Comment:
[EditorBrowsable(EditorBrowsableState.Never)]
Don't you need this?

Reply:
Thank you for your comment!
I have updated the code.